app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'user_id'
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
+
+  def latest_three_post
+    posts.order(created_at: :desc).limit(3)
+  end
+end
```

Comment:
- [ ] You are almost there! However, the validations are still missing for the `User` model:

- Name must not be blank.
- PostsCounter must be an integer greater than or equal to zero.

Please add them to this model.